InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using Microsoft.AspNetCore.Mvc.Testing;
+using RESTFulSense.Clients;
+
```

Comment:
Same here add line above and remove extras

Reply:
Made following changes.  let me know if any other changes needed.
1. add space after copyright (line 4)
2. remove space between single line statements (line 7)
3. remove space after closing braces (line 19)

![Screenshot (41)](https://user-images.githubusercontent.com/120409509/235209326-4899d33e-f1e8-4311-a20f-9daee083ac86.png)
@glhays